signin.js

```diff
@@ -26,32 +28,70 @@ function focusOutCheck(event){
   }
 }
 
+function passwordTypeToggle(event, id){
+  const $imageNode = document.querySelector(`#${event.target.id}`)
+  const $target = document.querySelector(`${id}`)
+  const targetType = $target.getAttribute('type'); // passwrod
+  $target.setAttribute('type', targetType === 'password' ? 'text' : 'password'); 
+  $imageNode.setAttribute('src',targetType === 'password' ? './images/eye-on.svg' : './images/eye-off.svg' )
+}
+
 
 function loginProcess() {
+ 
   const emailValue = document.querySelector('#email').value
   const passwordValue = document.querySelector('#password').value
-  if (emailValue !== 'test@codeit.com') {
-    showError('email', '이메일을 확인해 주세요.');
+
+  
+  if (!emailValue || !checkisValidEmail(emailValue)) {
```

Comment:
```suggestion
const isInValidEmail = !emailValue || !checkisValidEmail(emailValue);

  if (isInvalidEmail) {
```

위처럼 isInvalidEmail이라는 변수에 할당하면 코드를 읽을 때 바로 유효하지 않은 이메일이면 들어가는 문이라는 것을 명시적으로 이해할 수 있게 됩니다.

하지만 `!emailValue || !checkisValidEmail(emailValue)`과 같이 표현식을 적어두면 다른 사람 혹은 미래의 지영님께서 코드를 읽을 때 이 구문을 해석한 뒤에야 유효하지 않은 이메일인 경우에 들어가는 문이라고 인식할 수 있게 됩니다.

이처럼 변수를 필요에 따라 활용하면 가독성이 올라가는 효과가 있으니 사용해보시는 것을 권장드립니다.